app/templates/components/gh-image-uploader.hbs

```diff
@@ -19,10 +19,11 @@
                 <div class="description">{{description}}</div>
             {{/gh-file-input}}
         </div>
-
-        <a class="image-url" {{action "switchForm" "url-input"}}>
-            <i class="icon-link"><span class="hidden">URL</span></i>
-        </a>
+        {{#if allowUrlInput}}
```

Comment:
This may need testing with `fileStorage` being set to `false` in the config. If file storage is disabled we'll always display the url input form rather than the upload form - is that the desired behaviour?